src/items/repositories/item.repository.ts

```diff
@@ -326,4 +404,38 @@ export class ItemRepository extends Repository<Item> {
 
     return conditions;
   }
+
+  private getPaginationNftsConditions(
+    collectionId: string,
+    paginationDto: ItemPaginationDto
+  ): FindOptionsWhere<Item>[] {
+    const { priceFrom, priceTo, search } = paginationDto;
+    const conditions = [] as FindOptionsWhere<Item>[];
+    const condition1 = {
+      collection: new Collection(collectionId),
+      status: Not(ItemStatus.Hidden),
+    } as FindOptionsWhere<Item>;
+
+    if (priceFrom && priceTo) condition1.price = Between(String(priceFrom), String(priceTo));
+
+    const condition2 = { ...condition1 };
+
+    if (search) {
+      condition1.name = Raw(
+        (alias) => `LOWER(${alias}) Like '%${paginationDto.search?.toLowerCase()}%'`
+      );
+
+      condition2.description = Raw(
+        (alias) => `LOWER(${alias}) Like '%${paginationDto.search?.toLowerCase()}%'`
+      );
+    }
+
+    // This query will result on the following structure:
```

Comment:
Get rid of these comments

Reply:
These comments works to explain better what was done above